src/loan-options/FHA-loans/index.html

```diff
@@ -48,9 +48,9 @@ <h3 class="subhead tabbed-heading">Other loan types</h3>
       </header>
 
       <div class="aside-chunk">
-        <ul class="no-bullet">
-          <li><a class="go-link" href="../conventional-loans/">Conventional loans</a></li>
-          <li><a class="go-link" href="../special-loan-programs/">Special loan programs</a></li>
+        <ul class="list__unstyled">
+          <li><p><a class="go-link" href="../conventional-loans/">Conventional loans</a></p></li>
+          <li><p><a class="go-link" href="../special-loan-programs/">Special loan programs</a></p></li>
         </ul>
       </div>
 
```

Comment:
We can use some of the other list and link modifiers from [cf-typography](https://cfpb.github.io/cf-typography/docs/) to provide the icons & spacing/mobile stacking here. Maybe something like:

```html
<ul class="list list__unstyled list__links">
    <li class="list_item">
        <a class="list_link jump-link jump-link__right" href="#">
            <span class="jump-link_text">Conventional loans</span>
        </a>
   </li>
</ul>
```